pyairtable/utils.py

```diff
@@ -10,7 +10,9 @@ def datetime_to_iso_str(value: datetime) -> str:
     Args:
         value: datetime object
     """
-    return value.isoformat(timespec="milliseconds") + "Z"
+    if value.utcoffset() is None:
```

Comment:
Dont't we also need to check the `utcoffset`?
https://docs.python.org/3/library/datetime.html#determining-if-an-object-is-aware-or-naive